python/litestar/config.yaml

```diff
@@ -0,0 +1,8 @@
+framework:
+  website: litestar.dev
+  version: "2.0.0rc1"
+
+  engines:
+    - uvicorn
```

Comment:
⚠️ Actually results shows only the first engine, here `unicorn`. Is this the recommended way to deploy in **production** ?

Reply:
starlite is ASGI , unicorn is one of the "best" ASGI web server 

https://docs.litestar.dev/2/usage/applications.html#startup-and-shutdown

Reply:
and intern benchmark use unicorn -> https://docs.litestar.dev/2/benchmarks.html#methodology